checklist.html

```diff
@@ -482,68 +571,95 @@ <h2 id="label-forms">Forms</h2>
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-identified.html">3.3.1 Error Identification</a>
+        </p>
+        <p class="checklist-description">
+          Techniques such as <a href="https://developer.paciellogroup.com/blog/2018/09/describing-aria-describedby/">using <code>aria-describedby</code></a> allow people who use assistive technology to more easily understand the difference between the input and the error message associated with it.
+        </p>
       </div>
     </details>
     <details id="section-forms-06">
       <summary>
-        Ensure that error, warning, and success states are not visually communicated by just color.
+        Make sure that error, warning, and success states are not visually communicated by just color.
       </summary>
       <div class="checklist-details">
-        <p class="checklist-criterion"><a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html">1.4.1 Use of Color</a></p>
-        <p class="checklist-description">People who are color blind, who have other low vision conditions, or different cultural understandings for color may not see the state change, or understand what kind of feedback the state represents if color is the only indicator.</p>
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html">1.4.1 Use of Color</a>
+        </p>
+        <p class="checklist-description">
+          People who are color blind, who have other low vision conditions, or different cultural understandings for color may not see the state change, or understand what kind of feedback the state represents if color is the only indicator.
+        </p>
       </div>
     </details>
     <details id="section-forms-07">
       <summary>
         Update the page or view's <code>title</code> to include high-level form feedback.
       </summary>
       <div class="checklist-details">
-        <p class="checklist-criterion"><a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a></p>
-        <p class="checklist-description">The <code>title</code> element's content is often the first content announced. Adding feedback about the presence of things like errors or warnings allows people to know there are items on the page or view that need their attention.</p>
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a>
+        </p>
+        <p class="checklist-description">
+          The <code>title</code> element's content is often the first content announced by a screen reader. Adding feedback about the presence of things like errors or warnings allows people to know there are items on the page or view that need their attention.
```

Comment:
screen readers won't announce a change to the content of a `title` element.  So this would be more helpful for a sighted user, or if a full page reload/refresh were to occur, which would then cause the screen reader to announce the title again.